info/guide/diktat-coding-convention.md

```diff
@@ -1,3 +1,22 @@
+<img src="/logo.svg" width="64px"/>
+
+# Diktat Coding Convention
```

Comment:
This is old format of TOC, it's updated in master now. Please check and support new format in both generators from TOC.md to full guide md and then from full guide to tex.

Reply:
Added